en/assembly/quick_start_pixhawk4.md

```diff
@@ -155,13 +163,13 @@ General configuration information is covered in: [Autopilot Configuration](../co
 
 QuadPlane specific configuration is covered here: [QuadPlane VTOL Configuration](../config_vtol/vtol_quad_configuration.md)
 
-<!-- what about config of other vtol types and plane. Do the instructions in these ones above apply for tailsitters etc? -->
+<!-- what about config of other vtol types and plane. Do the instructions in these ones above apply for tailsitters etc? --> 
 
-<!--## Detailed Wiring Infographic (Copter)
+<!--## Detailed Wiring Infographic (Copter) 
 
-![QuadCopter Pixhawk Wiring Infographic](../../images/pixhawk_infographic2.jpg) -->
+![QuadCopter Pixhawk Wiring Infographic](../../images/pixhawk_infographic2.jpg) --> 
 
 ## Further information
-<!-- - [Pixhawk 4 Quick Start Guide (Holybro)]() Add link-->
+<!-- - [Pixhawk 4 Quick Start Guide (Holybro)]() Add link--> 

```

Comment:
Does this link exist for Holybrok quick start? When is it expected? If not, then we should make this link URL TBD, so at least we get a notification of error on build.